cli/command/trust/signer_add.go

```diff
@@ -44,9 +45,13 @@ func newSignerAddCommand(dockerCli command.Cli) *cobra.Command {
 	return cmd
 }
 
+var validSignerName = regexp.MustCompile(`^[a-zA-Z0-9\_]+[a-zA-Z0-9\_\-]*$`).MatchString
```

Comment:
let's also require that the signer name must be lowercase alphabetical characters only (remove `A-Z`) as this is a restriction of TUF